go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/go-plugin v1.4.3
 	github.com/hashicorp/go-retryablehttp v0.7.0
-	github.com/hashicorp/go-tfe v0.21.0
+	github.com/hashicorp/go-tfe v0.20.1-0.20211215223223-3057a43c4f5c
```

Comment:
Just a gut-check -- are we thinking to wait to merge this PR until the go-tfe changes have merged? Or will we treat this more like the run-up to 1.1.0, when we were building against a WIP branch for a while? 

If it's the latter, we should probably rebase the WIP branch and make sure we don't lose any features compared to what were previously at. It looks like with the current status, we might lose run variables?

Reply:
We need to have a slightly better process I think for pre-release go-tfe. I think we are going to wait on the go-tfe merge and depend directly on a branch until the next release of terraform that includes run tasks, similar to what we did in the run up to 1.1. Reason being, we want to make sure the API doesn't shift before releasing in go-tfe. This shouldn't compile if run variables weren't in the base, so it must be based on a run variables pre-release. I take your point and will make sure to rebase this to v0.22.0.